source/Mission.cpp

```diff
@@ -568,16 +568,34 @@ bool Mission::HasSpace(const PlayerInfo &player) const
 
 bool Mission::CanComplete(const PlayerInfo &player) const
 {
-	if(player.GetPlanet() != destination || !waypoints.empty() || !stopovers.empty())
+	if(player.GetPlanet() != destination)
 		return false;
 	
 	if(!toComplete.Test(player.Conditions()))
 		return false;
 	
+	if(!IsSatisfied(player))
+		return false;
+	
+	return true;
```

Comment:
It might be simpler and more readable to just `return IsSatisfied(player);` instead of the above four lines, just to avoid the "double negative" of "if not satisfied return false..."

Reply:
Addressed